style.css

```diff
@@ -83,6 +86,237 @@ body {
 
 /* End of section one(Headline) styling */
 
+/* BEGINNING OF MIDDLE SECTION STYLING */
+
+.article-img {
+  max-width: 100%;
+}
+
+.section-two {
+  display: grid;
+  background-color: #fff;
+  max-width: 300px;
+  margin: 5rem 1.5rem;
+  height: 600px;
+  padding: 1rem;
+  border-radius: 15px;
+}
+
+.article-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.info-div1 {
+  margin: 1rem 0;
+}
+
+.title {
+  color: #172b4d;
+  text-transform: capitalize;
+  font-size: 2rem;
+  font-family: poppins, sans-serif;
+}
+
+.info {
+  display: flex;
+  justify-content: space-between;
+  max-width: 250px;
+  margin: 1rem 0;
+}
+
+.canopy {
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  font-family: poppins, sans-serif;
+}
+
+.info-p {
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  font-family: poppins, sans-serif;
+}
+
+.profession {
+  text-transform: capitalize;
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: poppins, sans-serif;
+}
+
+.prof-span {
+  margin: 0 0.5rem;
+  color: #172b4d;
+}
+
+.tech-stack {
+  display: flex;
+  list-style-type: none;
+  margin: 1rem 0;
+  max-width: 200px;
+  justify-content: space-between;
+}
+
+.tech-stack li {
+  list-style-type: none;
+  border-radius: 5px;
+  background: #ebebff;
+  height: 22px;
+  max-width: 75px;
+  color: #6070ff;
+  margin-right: 0.5rem;
+  line-height: 16px;
+  font-size: 12px;
+  font-weight: 500px;
+  padding: 0 0.2rem;
+}
```

Comment:
- [ ] Please 🙏🏽 use the correct style, margin, padding, and font-family to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/blessing-michael/portfolio/assets/58642949/a876f6c8-7b87-4a68-92c8-e55f3f49cd79) | ![image](https://github.com/blessing-michael/portfolio/assets/58642949/42875fd4-21b1-47aa-8761-0b6529c7db5f) |